code/modules/items/specials/ItemSpecials.dm

```diff
@@ -1849,6 +1860,86 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 				playsound(master, 'sound/effects/swoosh.ogg', 50, FALSE)
 		return
 
+
+/datum/item_special/graffiti
+	cooldown = 30 //30
+	staminaCost = 5
+	moveDelay = 5
+	moveDelayDuration = 5
+
+	damageMult = 1
+
+	image = "swipe"
+	name = "Spray Burst"
+	desc = "Use a burst of spray paint to harmlessly blind, deface & disorient an opponent."
+	var/spray_color
+	/// If true, the swipe will ignite stuff in it's reach.
+	var/ignition = FALSE
+	var/flipped = FALSE
+
+	pixelaction(atom/target, list/params, mob/user, reach)
+		if(!isturf(target.loc) && !isturf(target)) return
+		if(!usable(user)) return
+		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

Comment:
Could probably do an early return here to remove need of indenting rest of function?